crates/primitives/src/chain.rs

```diff
@@ -138,13 +184,19 @@ impl Chain {
             | Self::ZkSync
             | Self::Linea
             | Self::Mantle
-            | Self::Celo => 60,
+            | Self::Celo
+            | Self::Dymension => 60,
```

Comment:
that's unusual for cosmos chain to pick `60`

Reply:
dimension has ETH address too, that's why it uses 60, similar to Evmos and injective